src/styles/utils/_media.scss

```diff
@@ -0,0 +1,17 @@
+@media screen and (max-width: 1023px) {
+  .hero {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media screen and (max-width: 767px) {
+  .hero {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media screen and (max-width: 487px) {
+  .hero {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
```

Comment:
This works good, but you typically want to style with [mobile-first approach](https://zellwk.com/blog/how-to-write-mobile-first-css/).